### ISSUE TITLE
Unhealthy status is not reset

### ISSUE THREAD

**Issue by Zelldon** [zeebe-io/zeebe#6393]
**Description**

[It looks like when the broker becomes unhealthy because of out of disk space](https://console.cloud.google.com/logs/viewer?interval=NO_LIMIT&authuser=1&project=zeebe-io&minLogLevel=0&expandAll=false&timestamp=2021-02-19T13:58:05.165000000Z&customFacets=&limitCustomFacetWidth=true&advancedFilter=resource.type%3D%22k8s_container%22%0Aresource.labels.project_id%3D%22zeebe-io%22%0Aresource.labels.location%3D%22europe-west1-b%22%0Aresource.labels.cluster_name%3D%22zeebe-cluster%22%0Aresource.labels.namespace_name%3D%22zell-kw-06-benchmark%22%0Alabels.k8s-pod%2Fapp_kubernetes_io%2Fcomponent%3D%22broker%22%0Alabels.k8s-pod%2Fapp_kubernetes_io%2Finstance%3D%22zell-kw-06-benchmark%22%0Alabels.k8s-pod%2Fapp_kubernetes_io%2Fmanaged-by%3D%22Helm%22%0Alabels.k8s-pod%2Fapp_kubernetes_io%2Fname%3D%22zeebe-cluster%22%0A%22unhealthy%22&scrollTimestamp=2021-02-19T13:35:06.021062000Z&pinnedLogId=kgtv3fw5i7j2&pinnedLogTimestamp=2021-02-18T19:37:14.272804Z) and it recovers it will not become healthy again.

![healthl](https://user-images.githubusercontent.com/2758593/108513294-ab69ed80-72c2-11eb-8bb9-bf6707da3b19.png)

![general](https://user-images.githubusercontent.com/2758593/108513292-aad15700-72c2-11eb-9990-dc579468fdf0.png)

![elastic](https://user-images.githubusercontent.com/2758593/108513284-a9a02a00-72c2-11eb-922c-248fec0210c7.png)
![disc](https://user-images.githubusercontent.com/2758593/108513288-aad15700-72c2-11eb-98ab-d4d5ee40c145.png)

[Around ~ 4 am it is able to make process again. Elastic is writable again.](https://console.cloud.google.com/logs/viewer?interval=JUMP_TO_TIME&authuser=1&project=zeebe-io&minLogLevel=0&expandAll=false&timestamp=2021-02-19T03:10:15.459000000Z&customFacets=&limitCustomFacetWidth=true&advancedFilter=resource.type%3D%22k8s_container%22%0Aresource.labels.project_id%3D%22zeebe-io%22%0Aresource.labels.location%3D%22europe-west1-b%22%0Aresource.labels.cluster_name%3D%22zeebe-cluster%22%0Aresource.labels.namespace_name%3D%22zell-kw-06-benchmark%22%0Alabels.k8s-pod%2Fapp_kubernetes_io%2Fcomponent%3D%22broker%22%0Alabels.k8s-pod%2Fapp_kubernetes_io%2Finstance%3D%22zell-kw-06-benchmark%22%0Alabels.k8s-pod%2Fapp_kubernetes_io%2Fmanaged-by%3D%22Helm%22%0Alabels.k8s-pod%2Fapp_kubernetes_io%2Fname%3D%22zeebe-cluster%22&scrollTimestamp=2021-02-19T03:02:43.630561000Z&dateRangeStart=2021-02-19T02:10:15.459Z&dateRangeEnd=2021-02-19T04:10:15.459Z&pinnedLogId=10mk1wefslrwf5&pinnedLogTimestamp=2021-02-19T03:00:15.315593Z)
```
W 2021-02-19T03:00:14.127455Z Error on exporting record with key 6755400044295841 
W 2021-02-19T03:00:15.315177Z Failed to flush 1000 item(s) of bulk request [type: cluster_block_exception, reason: blocked by: [FORBIDDEN/12/index read-only / allow delete (api)];] 
E 2021-02-19T03:00:15.315593Z Unexpected exception occurred on periodically flushing bulk, will retry later. 
D 2021-02-19T03:02:42.889149Z Based on lowest exporter position '1400625023' and last processed position '1410610457', determined '1400625023' as snapshot position. 
D 2021-02-19T03:02:43.450737Z Taking temporary snapshot into /usr/local/zeebe/data/raft-partition/partitions/3/pending/1134464099-21-1613703763450-1410610457-1400625023. 
D 2021-02-19T03:02:43.534455Z Created snapshot for Broker-2-StreamProcessor-3 
```

**Comment by npepinpe** [comment]
Duplicate of #6232
